docs/design/features/globalization-hybrid-mode.md

```diff
@@ -408,4 +408,34 @@ Affected public APIs:
 - CompareInfo.GetSortKeyLength
 - CompareInfo.GetHashCode
 
-Apple Native API does not have an equivalent, so they throw `PlatformNotSupportedException`.
\ No newline at end of file
+Apple Native API does not have an equivalent, so they throw `PlatformNotSupportedException`.
+
+
+## Case change
+
+Affected public APIs:
+- TextInfo.ToLower,
+- TextInfo.ToUpper
+
+Below function are used from apple native functions:
+- [uppercaseString](https://developer.apple.com/documentation/foundation/nsstring/1409855-uppercasestring)
+- [lowercaseString](https://developer.apple.com/documentation/foundation/nsstring/1408467-lowercasestring)
+- [uppercaseStringWithLocale](https://developer.apple.com/documentation/foundation/nsstring/1413316-uppercasestringwithlocale?language=objc)
+- [lowercaseStringWithLocale](https://developer.apple.com/documentation/foundation/nsstring/1417298-lowercasestringwithlocale?language=objc)
+
+Behavioural changes compared to ICU
+
+   - Final sigma behavior correction:
+
+     ICU-based case change does not respect final-sigma rule, but hybrid does, so "ΒΌΛΟΣ" -> "βόλος", not "βόλοσ".
+
+   - Below cases will throw exception because of insufficiently sized destination buffer
```

Comment:
If we don't plan to expand the buffer, then we need a similar workaround as we have for WASM: https://github.com/dotnet/runtime/pull/86799. We are keeping the lower ß when upper casing. It is more costly but throwing depending on the string contents in limiting the usage of the function a bit too much - the developer cannot asses if they can use our Hybrid because they cannot know what strings will enter the ToUpper function.

Reply:
This looks quite costly. I opened a separate ticket to investigate it further https://github.com/dotnet/runtime/issues/88180